_posts/2012-11-16-november-2012.md

```diff
@@ -20,13 +20,13 @@ To support you in collecting and reporting your site [search metrics]({{ site.ba
 1. **Top changing search terms (movement up/down).** Shows trending topics, what's hot or not. Use this report to investigate why people might be losing interest in once-popular content and consider archiving if it's no longer needed. For newly popular terms, create new content or update existing content to ensure it's current, accurate, and complete. 
 1. **Top clicked URLs for specific queries.** The most common results that searchers found most relevant or appealing, for specific search terms. Review these popular pages and other customer top tasks to make sure they are current, accurate, complete and compelling.
 
-### 2. Recommended Pages in Type-ahead Suggestions
+### 2. Recommended Pages in typeahead Suggestions
 
 Searchers often know the exact page they're looking for. We've added a feature to help them find that page even more quickly. 
 
 As searchers type in their query, relevant [text Best Bets]({{ site.baseurl }}/admin-center/content/best-bets.html) are shown as Recommended Pages, as shown in the sample below for a search on [*fa&#8230;* on DOL.gov](https://search.usa.gov/search?query=jobs&amp;affiliate=u.s.departmentoflabor).
 
-![Type-ahead suggestions on DOL.gov's search results  page](https://d3qcdigd1fhos0.cloudfront.net/blog/img/tumblr_mdljlrljT71qid15q.png)
+![typeahead suggestions on DOL.gov's search results  page](https://d3qcdigd1fhos0.cloudfront.net/blog/img/tumblr_mdljlrljT71qid15q.png)
```

Comment:
Same comment here, should be title-case. I think we're deleting this anyway though with the other PR.

Reply:
Yes, and this file was deleted with the other PR, so I didn't bother cleaning it up after the global edit.